wrapper/fmriprep_docker.py

```diff
@@ -231,6 +232,9 @@ def main():
                         type=os.path.abspath,
                         help='Grid reference image for resampling BOLD files to volume template '
                              'space.')
+    g_wrap.add_argument('--fs-license-file', metavar='PATH', type=os.path.abspath,
+                        default=os.getenv('FS_LICENSE', None),
+                        help='folder containing FreeSurfer\'s license.txt file')
```

Comment:
`help="Path to FreeSurfer license key file. To obtain it you need to register (for free) at https://surfer.nmr.mgh.harvard.edu/registration.html"`